espymetrics/app.py

```diff
@@ -9,7 +9,7 @@
 from clastic.errors import BadRequest
 from clastic.middleware import GetParamMiddleware
 
-from dal import LineDAL
+from dal import LineDAL, SQLiteDAL
```

Comment:
This is a good change.

**NB**: This is the only affirmative comment. All the comments inline below are critiques. If they make the antipattern code a bit hard to follow, feel free to use the "View" button in the top right of the sqlite_dal.py file below.